packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -0,0 +1,88 @@
+import { LitElement, customElement, html, property } from 'lit-element';
+// import { Base } from '@sfx/base';
+import '@sfx/autocomplete';
```

Comment:
I've wondered about this for the others, but is it really the component's responsibility to be pulling this in?